default.ps1

```diff
@@ -78,7 +78,7 @@ task Nuget {
       @("RhinoDSL", "1.0.0"), `
       @("Common.Logging", "3.0.0"), `
       @("Common.Logging.Core", "3.0.0"), `
-      @("FileHelpers", "2.0.0.0") `
+      @("FileHelpers", "3.0.46-beta") `
```

Comment:
Hi there, we are planning to release 3.1 final version in the next two weeks, we released 3.1-rc now, the binaries will be near the same because we are no planning changes, only finishing the new site and docs in www.filehelpers.net

I will send a new pull request after we release final 3.1
Best Regards